src/BuiltInTools/dotnet-watch/Filters/BrowserRefreshFilter.cs

```diff
@@ -38,6 +40,13 @@ public async ValueTask ProcessAsync(DotNetWatchContext context, CancellationToke
                     _reporter.Verbose("Unable to determine if this project is a webapp.");
                     return;
                 }
+                else if (!IsSupportedVersion(context.ProjectGraph))
+                {
+                    _reporter.Warning(
+                        "Skipping configuring browser-refresh middleware since the target framework version is not supported." +
+                        " For more information see 'https://aka.ms/dotnet/watch/unsupported-version'.");
```

Comment:
@mkArtakMSFT does this URL work for you (we'll have to create it and point it to the docs).

Reply:
Thanks @javiercn . Can you please change the link to this one instead:
https://aka.ms/dotnet/watch/unsupported-tfm

Reply:
```suggestion
                        " For more information see 'https://aka.ms/dotnet/watch/unsupported-tfm'.");
```